modules/local/virsorter2/main.nf

```diff
@@ -0,0 +1,38 @@
+process VIRSORTER2 {
+    tag "$meta.id"
+    label 'process_medium'
+    conda "bioconda::virsorter=2.2.3-pyhdfd78af_1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://jiarong/virsorter:2.2.3' :
+        'jiarong/virsorter:2.2.3' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    path "${prefix}/final-viral-boundary.tsv" , emit: boundary                                     
+    path "${prefix}/final-viral-combined.fa"  , emit: fasta
+    path "${prefix}/final-viral-score.tsv"    , emit: score
+    path "versions.yml"                       , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args     = task.ext.args ?: ''
+    def db       = params.virsorter_db ?: '/db'
```

Comment:
check if input ch_db==null then db = /db
else: db==ch_db

Reply:
resolved in https://github.com/cidgoh/nf-seqqc/pull/52/commits/30067fb3b5f244ca93b80296dce93214cc55c1b6